packages/frontend-usagers/src/components/organisme/personne-morale.vue

```diff
@@ -400,6 +426,7 @@ async function searchOrganisme() {
 
 function next() {
   log.i("next - IN");
+  log.i({ ...values });
```

Comment:
log.d ou log.i("next - IN", { ...values })

Reply:
Done